ubuntu.22.04/Dockerfile

```diff
@@ -58,7 +58,10 @@ RUN wget --quiet -O - https://raw.githubusercontent.com/kubernetes/helm/master/s
 
 # Get .NET SDK 6.0
 # https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1804
+# https://learn.microsoft.com/en-us/dotnet/core/install/linux-package-mixup
 RUN DOTNET_CLI_TELEMETRY_OPTOUT=1 && \
+    touch /etc/apt/preferences && \

```

Comment:
We need to ignore DotNet related packages from `packages.microsoft.com`, which now double up with the actual Ubuntu apt releases.